packages/grpc-authentication/src/index.ts

```diff
@@ -8,20 +8,41 @@ import { Client } from '@textile/hub-threads-client'
  * Set the options for authenticating with an API key
  */
 export interface WithKeyInfoOptions extends WithUserAuthOptions {
+  /**
+   * Optional. Get more detailed log information.
+   */
+  debug?: boolean
+  /**
+   * Optional. Future date for the session to expire. Short time periods are strongly recommended. (Default 30 minutes).
+   */
   date?: Date
+  /**
+   * Development purposes only.
+   */
+  host?: string
```

Comment:
same here with `host` and `debug` properties being implied

Reply:
same yeah